File: MP1/tempConv.ts
function fahrenheitToCelsius(temp: number): number {
    return (temp - 32) * 5 / 9;
}

const input = prompt("Input temperature in Fahrenheit (F): ");
console.log(`Converted to Celsius (C): ${fahrenheitToCelsius(Number(input)).toFixed(6)}`);

File: MP1/tempConv.c
#include <stdio.h>

float FahrenheitToCelsius(float temp);

int main(){
    
    float temp;
    printf("Input temperature in Fahrenheit(F): ");
    scanf("%f", &temp);
    printf("Converted to Celsius (C): %f", FahrenheitToCelsius(temp));

return 0;    
};


float FahrenheitToCelsius(float temp){
    return (temp - 32) * 5 /9; 
};
